native-src/android/app/src/main/java/com/telerik/pushplugin/PushPluginListener.java

```diff
@@ -10,13 +10,12 @@
      * Defines a success callback method, which is used to pass success function reference
      * from the nativescript to the Java plugin
      *
-     * @param message
      * @param data
+     * @param notification
      */
-    void success(Object message, Object data, RemoteMessage.Notification notificationObj);
-    void success(Object message, Object data);
-    void success(Object message); // method overload to mimic optional argument
-
+    void success(Object data, RemoteMessage.Notification notification);

```

Comment:
We should also update the push plugin sample app to reflect these breaking changes since it uses the latest version of the plugin and won't work properly after this is released.